commonjs/angular-numbro.min.js

```diff
@@ -0,0 +1,10 @@
+window.require.define({"angular-numbro.min": function(exports, require, module) {
```

Comment:
Line is too long.<br>'module' is defined but never used.<br>'require' is defined but never used.<br>'exports' is defined but never used.